src/notation/view/selectionfiltermodel.cpp

```diff
@@ -205,9 +229,34 @@ QString SelectionFilterModel::titleForType(SelectionFilterType type) const
         return muse::qtrc("notation", "Tremolos");
     case SelectionFilterType::GRACE_NOTE:
         return muse::qtrc("notation", "Grace notes");
-    case SelectionFilterType::NONE:
-        break;
+    default: break;
```

Comment:
(A disadvantage of adding a `default` here, is that we won't get a compiler warning anymore when we add a new case to `SelectionFilterType` and forget to handle it here; but perhaps that won't happen anyway)